x-pack/plugins/cases/server/client/cases/mock.ts

```diff
@@ -228,6 +228,16 @@ export const commentPersistableState: Comment = {
   version: 'WzEsMV0=',
 };
 
+export const commentFilePersistableState: Comment = {
```

Comment:
As this is an external reference attachment and not a persistable state maybe it is better to rename it to avoid confusion.